docs/content/http-proxy.md

```diff
@@ -34,7 +34,7 @@ spec:
       - name: proxy
         values:
           http: http://example.com
-          https: http://example.com
+          https: https://example.com
```

Comment:
Nit: you can have http endpoint for https proxy

Reply:
Good point, reverted.